src/test/OpStateBridge.t.sol

```diff
@@ -314,27 +238,12 @@ contract StateBridgeTest is PRBTest, StdCheats {
         vm.expectRevert("Ownable: caller is not the owner");
 
         vm.prank(nonOwner);
-        stateBridge.transferOwnership(newOwner);
-    }
-
-    /// @notice tests that the StateBridge contract's ownership can't be changed by a non-owner
-    /// @param newOwner The new owner of the StateBridge contract (foundry fuzz)
-    function test_notOwner_transferOwnershipOptimism_reverts(
-        address nonOwner,
-        address newOwner,
-        bool isLocal
-    ) public {
-        vm.assume(nonOwner != owner && newOwner != address(0x0));
-
-        vm.expectRevert("Ownable: caller is not the owner");
-
-        vm.prank(nonOwner);
-        stateBridge.transferOwnershipOptimism(newOwner, isLocal);
+        opStateBridge.transferOwnership(newOwner);
     }
 
     /// @notice tests that the StateBridge contract's ownership can't be changed by a non-owner
     /// @param newOwner The new owner of the StateBridge contract (foundry fuzz)
-    function test_notOwner_transferOwnershipBase_reverts(
+    function test_notOwner_transferOwnershipOp_reverts(
```

Comment:
We should also have test for failure cases when the `newOwner == address(0)`.

Reply:
you were right, after testing, `Ownable2Step` does allow to update `owner` to the zero address.

Reply:
reached out to OZ with a question